tests/libs/datasets/custom_aggregations_test.py

```diff
@@ -105,17 +104,11 @@ def test_aggregate_to_country():
 
     # State data has some hospital and cases time series.
     states_timeseries = {
-        region_il: {
-            CommonFields.ALL_BED_TYPICAL_OCCUPANCY_RATE: [0.5, 0.5],
-            CommonFields.CASES: [1, 2],
-        },
-        region_tx: {
-            CommonFields.ALL_BED_TYPICAL_OCCUPANCY_RATE: [0.7, 0.7],
-            CommonFields.CASES: [0, 3],
-        },
+        region_il: {CommonFields.CURRENT_HOSPITALIZED: [5, 5], CommonFields.CASES: [1, 2],},
+        region_tx: {CommonFields.CURRENT_HOSPITALIZED: [7, 7], CommonFields.CASES: [0, 3],},
```

Comment:
I switched this test from testing the aggregation of ALL_BED_TYPICAL_OCCUPANCY_RATE (now removed) to CURRENT_HOSPITALIZED. 🤷